### ISSUE TITLE
dyld: Library not loaded: /usr/local/opt/pcre2/lib/libpcre2-8.0.dylib

### ISSUE THREAD

**Issue by lchen91** [JuliaDocs/Documenter.jl#899]
I am using Julia v0.7 on OSX 10.11.6.

When I run `julia make.jl`, I got the following message. I tried `brew reinstall pcre2` but it didn't work.
```
 dyld: Library not loaded: /usr/local/opt/pcre2/lib/libpcre2-8.0.dylib
  Referenced from: /usr/local/bin/git
  Reason: Incompatible library version: git requires version 8.0.0 or later, but libpcre2-8.0.dylib provides version 7.0.0
Documenter: setting up build directory.
Documenter: expanding markdown templates.
Documenter: building cross-references.
Documenter: running document checks.
 > checking for missing docstrings.
 > running doctests.
 > checking footnote links.
Documenter: populating indices.
Documenter: rendering document.
dyld: Library not loaded: /usr/local/opt/pcre2/lib/libpcre2-8.0.dylib
  Referenced from: /usr/local/bin/git
  Reason: Incompatible library version: git requires version 8.0.0 or later, but libpcre2-8.0.dylib provides version 7.0.0
dyld: Library not loaded: /usr/local/opt/pcre2/lib/libpcre2-8.0.dylib
  Referenced from: /usr/local/bin/git
  Reason: Incompatible library version: git requires version 8.0.0 or later, but libpcre2-8.0.dylib provides version 7.0.0
dyld: Library not loaded: /usr/local/opt/pcre2/lib/libpcre2-8.0.dylib
  Referenced from: /usr/local/bin/git
  Reason: Incompatible library version: git requires version 8.0.0 or later, but libpcre2-8.0.dylib provides version 7.0.0
ERROR: LoadError: failed process: Process(`git rev-parse HEAD`, ProcessSignaled(5)) [0]
Stacktrace:
 [1] error(::String, ::Base.Process, ::String, ::Int64, ::String) at ./error.jl:42
 [2] pipeline_error at ./process.jl:712 [inlined]
 [3] read(::Cmd) at ./process.jl:648
 [4] read(::Cmd, ::Type{String}) at ./process.jl:652
 [5] readchomp(::Cmd) at ./io.jl:797
 [6] #5 at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Utilities/Utilities.jl:385 [inlined]
 [7] cd(::getfield(Documenter.Utilities, Symbol("##5#6")), ::String) at ./file.jl:96
 [8] repo_commit at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Utilities/Utilities.jl:384 [inlined]
 [9] url(::String, ::String, ::Module, ::String, ::UnitRange{Int64}) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Utilities/Utilities.jl:440
 [10] url(::String, ::String, ::Base.Docs.DocStr) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Utilities/Utilities.jl:405
 [11] (::getfield(Documenter.Writers.HTMLWriter, Symbol("##62#63")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode,Documenter.Utilities.DOM.Tag})(::Tuple{Markdown.MD,Base.Docs.DocStr}) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:821
 [12] collect(::Base.Generator{Base.Iterators.Zip2{Array{Any,1},Array{Base.Docs.DocStr,1}},getfield(Documenter.Writers.HTMLWriter, Symbol("##62#63")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode,Documenter.Utilities.DOM.Tag}}) at ./generator.jl:47
 [13] map(::Function, ::Base.Iterators.Zip2{Array{Any,1},Array{Base.Docs.DocStr,1}}) at ./abstractarray.jl:2021
 [14] domify_doc(::Documenter.Writers.HTMLWriter.HTMLContext, ::Documenter.Documents.NavNode, ::Markdown.MD) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:816
 [15] domify(::Documenter.Writers.HTMLWriter.HTMLContext, ::Documenter.Documents.NavNode, ::Documenter.Documents.DocsNode) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:799
 [16] #60 at ./none:0 [inlined]
 [17] iterate at ./generator.jl:47 [inlined]
 [18] collect(::Base.Generator{Array{Documenter.Documents.DocsNode,1},getfield(Documenter.Writers.HTMLWriter, Symbol("##60#61")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode}}) at ./array.jl:619
 [19] domify(::Documenter.Writers.HTMLWriter.HTMLContext, ::Documenter.Documents.NavNode, ::Documenter.Documents.DocsNodes) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:785
 [20] (::getfield(Documenter.Writers.HTMLWriter, Symbol("##54#55")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode,Documenter.Documents.Page,Documenter.Writers.HTMLWriter.SearchIndexBuffer})(::Markdown.Code) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:616
 [21] collect_to!(::Array{Documenter.Utilities.DOM.Node,1}, ::Base.Generator{Array{Any,1},getfield(Documenter.Writers.HTMLWriter, Symbol("##54#55")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode,Documenter.Documents.Page,Documenter.Writers.HTMLWriter.SearchIndexBuffer}}, ::Int64, ::Int64) at ./generator.jl:47
 [22] collect_to_with_first!(::Array{Documenter.Utilities.DOM.Node,1}, ::Documenter.Utilities.DOM.Node, ::Base.Generator{Array{Any,1},getfield(Documenter.Writers.HTMLWriter, Symbol("##54#55")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode,Documenter.Documents.Page,Documenter.Writers.HTMLWriter.SearchIndexBuffer}}, ::Int64) at ./array.jl:643
 [23] _collect(::Array{Any,1}, ::Base.Generator{Array{Any,1},getfield(Documenter.Writers.HTMLWriter, Symbol("##54#55")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode,Documenter.Documents.Page,Documenter.Writers.HTMLWriter.SearchIndexBuffer}}, ::Base.EltypeUnknown, ::Base.HasShape{1}) at ./array.jl:637
 [24] collect_similar(::Array{Any,1}, ::Base.Generator{Array{Any,1},getfield(Documenter.Writers.HTMLWriter, Symbol("##54#55")){Documenter.Writers.HTMLWriter.HTMLContext,Documenter.Documents.NavNode,Documenter.Documents.Page,Documenter.Writers.HTMLWriter.SearchIndexBuffer}}) at ./array.jl:561
 [25] map(::Function, ::Array{Any,1}) at ./abstractarray.jl:1995
 [26] domify(::Documenter.Writers.HTMLWriter.HTMLContext, ::Documenter.Documents.NavNode) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:614
 [27] render_article(::Documenter.Writers.HTMLWriter.HTMLContext, ::Documenter.Documents.NavNode) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:493
 [28] render_page(::Documenter.Writers.HTMLWriter.HTMLContext, ::Documenter.Documents.NavNode) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:204
 [29] render(::Documenter.Documents.Document) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/HTMLWriter.jl:152
 [30] runner(::Type{Documenter.Writers.HTMLFormat}, ::Symbol, ::Documenter.Documents.Document) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/Writers.jl:43
 [31] dispatch(::Type{Documenter.Writers.FormatSelector}, ::Symbol, ::Vararg{Any,N} where N) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Utilities/Selectors.jl:167
 [32] macro expansion at ./logging.jl:309 [inlined]
 [33] render(::Documenter.Documents.Document) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Writers/Writers.jl:66
 [34] runner(::Type{Documenter.Builder.RenderDocument}, ::Documenter.Documents.Document) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Builder.jl:207
 [35] dispatch(::Type{Documenter.Builder.DocumentPipeline}, ::Documenter.Documents.Document) at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Utilities/Selectors.jl:167
 [36] #2 at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Documenter.jl:191 [inlined]
 [37] cd(::getfield(Documenter, Symbol("##2#3")){Documenter.Documents.Document}, ::String) at ./file.jl:96
 [38] #makedocs#1 at /Users/linchen/.julia/packages/Documenter/e77Rr/src/Documenter.jl:190 [inlined]
 [39] (::getfield(Documenter, Symbol("#kw##makedocs")))(::NamedTuple{(:sitename, :repo),Tuple{String,String}}, ::typeof(makedocs)) at ./none:0
 [40] top-level scope at none:0
 [41] include at ./boot.jl:317 [inlined]
 [42] include_relative(::Module, ::String) at ./loading.jl:1038
 [43] include(::Module, ::String) at ./sysimg.jl:29
 [44] exec_options(::Base.JLOptions) at ./client.jl:239
 [45] _start() at ./client.jl:432
in expression starting at /Users/linchen/Documents/YaleResearch/Submodular_Utilities/docs/make.jl:4
```
Please let me know if additional information is needed. It would be great if I could have some advice. Thank you!

**Comment by fredrikekre** [comment]
Why do you think this is an issue with Documenter?

**Comment by mortenpi** [comment]
Yeah, it looks like a problem with your `git` installation and not related to Documenter (except for that Documenter needs to run the `git` binary).

**Comment by lchen91** [comment]
Thank you both. I fixed it. It turns out not to be an issue of
Documenter.jl.

On December 7, 2018 at 7:44:54 PM, Morten Piibeleht (
notifications@github.com) wrote:

Closed #899 <https://github.com/JuliaDocs/Documenter.jl/issues/899>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/JuliaDocs/Documenter.jl/issues/899#event-2013725353>,
or mute the thread
